README.md

```diff
@@ -117,14 +120,14 @@ issue][issue].
 
 ### Docker Hub rate limit
 
-We're planning to release v0.20.0 with a better user interface. Until then, this is the usable workaround.
+We're planning a future release with a better user interface. Until then, this is the usable workaround.
```

Comment:
Possibly add a tidbit about Docker Hub rate limiting... "This year, Docker has started rate limiting image pulls from Docker hub. This release introduces a workaround when using Sonobuoy (future release will come with a better user experience" or something similar to that (sorry for all these words).